Makefile

```diff
@@ -164,6 +168,14 @@ auraed-test: musl proto
 auraed-test-all: musl proto
 	@sudo -E $(cargo) test --target $(uname_m)-unknown-linux-musl -p auraed -- --include-ignored
 
+.PHONY: auraed-build
+auraed-build: musl proto
```

Comment:
Would it be beneficial to combine `musl` and `proto` into an `init` step or something?

Reply:
probably not. we can, but it's better to use targets as "build tasks" and we wouldn't (I think) run musl+proto without doing other things.